package.json

```diff
@@ -47,6 +47,7 @@
         "@types/node": "^12.7.2",
         "core-js": "^2.6.9",
         "focus-trap": "^5.0.2",
+        "fundamental-ngx": "^0.9.0",
```

Comment:
Please remove it and rebuild package-lock.json.

Reply:
I would say revert the change